durabletask/worker.py

```diff
@@ -778,6 +838,7 @@ def call_sub_orchestrator(
             retry_policy=retry_policy,
             is_sub_orch=True,
             instance_id=instance_id,
+            version=version,
```

Comment:
Need to find out how to get the default version from worker and use it here if version from the params is not specified